cpp/CMakeLists.txt

```diff
@@ -236,6 +236,7 @@ if(BUILD_CUML_CPP_LIBRARY)
   # FIL components
   target_sources(${CUML_CPP_TARGET}
     PRIVATE
+      src/fil/cudf_to_matrix.cu
```

Comment:
It think it would be better to place this in some more generic directory, as the functionality isn't directly tied to FIL.